src/logging.c

```diff
@@ -86,6 +87,54 @@ rcutils_ret_t rcutils_logging_initialize(void)
   return rcutils_logging_initialize_with_allocator(rcutils_get_default_allocator());
 }
 
+enum rcutils_get_env_retval
+{
+  RCUTILS_GET_ENV_ERROR = -1,
+  RCUTILS_GET_ENV_ZERO = 0,
+  RCUTILS_GET_ENV_ONE = 1,
+  RCUTILS_GET_ENV_DEFAULT = 2,
+};
+
+// A utility function to get zero or one from an environment variable.  Returns
+// RCUTILS_GET_ENV_ERROR if we failed to get the environment variable or if it was
+// something we don't understand.  Return RCUTILS_GET_ENV_ZERO if the value in the
+// environment variable is "0", RCUTILS_GET_ENV_ONE if the value in the environment
+// variable is "1", or RCUTILS_GET_ENV_DEFAULT if the environment variables is empty.
+static enum rcutils_get_env_retval rcutils_get_env_var_zero_or_one(
```

Comment:
@clalancette nit: any reason not to use `Doxygen` syntax?

Reply:
> @clalancette nit: any reason not to use `Doxygen` syntax?

Not particularly.  This is an internal function, so I didn't think it was necessary, but I'll claim ignorance on what our policy is for comments on internal functions.